Bottom: 80be38578cf732a8bb7a233550deccf5018838d9
Top:    a0c8a65d0f22056b271bb45c28bc09d8dd7fde77
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-29 10:40:36 +0800

Refresh of tun-allow-attach-ebpf-filter

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index fdd90f5..af85290 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -917,7 +917,6 @@ static int run_filter(struct tap_filter *filter, const struct sk_buff *skb)
 }
 
 
-
 /*
  * Checks whether the packet is accepted or not.
  * Returns: 0 - drop, !=0 - accept
